src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/forrigebehandling/EndringIUtbetalingUtilTest.kt

```diff
@@ -0,0 +1,195 @@
+package no.nav.familie.ks.sak.kjerne.forrigebehandling
+
+import no.nav.familie.ks.sak.common.util.tilYearMonth
+import no.nav.familie.ks.sak.data.lagAndelTilkjentYtelse
+import no.nav.familie.ks.sak.data.lagPerson
+import no.nav.familie.ks.sak.data.randomAktør
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonType
+import no.nav.familie.tidslinje.utvidelser.tilPerioder
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import java.time.YearMonth
+
+class EndringIUtbetalingUtilTest {
+    val jan22 = YearMonth.of(2022, 1)
+    val aug22 = YearMonth.of(2022, 8)
+    val sep22 = YearMonth.of(2022, 9)
+    val des22 = YearMonth.of(2022, 12)
```

Comment:
Disse kan kanskje være `private`? 

```suggestion
    private val jan22 = YearMonth.of(2022, 1)
    private val aug22 = YearMonth.of(2022, 8)
    private val sep22 = YearMonth.of(2022, 9)
    private val des22 = YearMonth.of(2022, 12)
```